docs/topics/api/collections.rst

```diff
@@ -58,6 +65,11 @@ If your account has the `Collections:Edit` permission then you can access any co
     :>json string uuid: A unique identifier for this collection; primarily used to count addon installations that come via this collection.
 
 
+If the ``with_addon`` parameter is passed then :ref:`addons in the collection<collection-addon-detail>` are returned along with the detail.
```

Comment:
Could you document how that looks in the response? I assume it will just add an `addons` key to the detail object? The pattern the docs usually follow is something like:

Response JSON Object:

- **id**: ...
- **addon_count**: ...
- **addons**: (null|array) Some description here and a link to the add-on object.